### PR TITLE
Add storybook addon to show components in RTL

### LINKED ISSUE #328: Investigate how we add RTL/LTR Support to components
**Is your feature request related to a problem? Please describe.**
We need to be able to test how components react to LTR/RTL text direction.

**Describe the solution you'd like**
We want to install Storybook Addon RTL to toggle the text direction per component.

**Describe alternatives you've considered**
N/A

**Testing notes**
N/A

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc
N/A

**Additional context**
https://github.com/unindented/storybook-addon-rtl

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by j-pendlebury**
Resolves #328

Alternative solution to https://github.com/bbc/psammead/pull/335.

**Overall change:**
We (the WS team) need to make sure that the current components will work with RTL left scripts. This is the beginning of such work.

**Code changes:**

- Installed and configured the [RTL Storybook Addon](https://www.npmjs.com/package/storybook-addon-rtl) to test components with RTL text

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [x] Test engineer approval

### CODE REVIEW

**Review by dr3 — CHANGES_REQUESTED** [review]
LGTM :) Seeing as youve bumped the main psammead version (🎉 ) can you make a changelog please?

Also can we go to psammead v1.0.0 or 1.1.0? I think it will give us more freedom in our versioning

**Comment by j-pendlebury** [comment]
@dr3 I knew I was forgetting something! 😄 the CHANGELOG now exists, but I only added the new version and the changes I've made. Not sure if you guys wanna add more history to it...

**Review by dr3 — DISMISSED** [review]

**Review by andrew-nowak — DISMISSED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead",
-  "version": "0.1.0",
+  "version": "1.1.0",
```
Comment (pjlee11) [inline package.json:3]: In the [Semver syntax](https://semver.org/) this should change to `1.0.0`. However, I spoke with @dr3 IRL and we've agreed that this should have already been `1.0.0` so it makes sense for this PR to bump to `1.1.0`. 

Adding this for future reference of npm versions.

**Review by pjlee11 — DISMISSED** [review]
LGTM 👍

**Review by sareh — DISMISSED** [review]
👍 Looks great.

**Review by dr3 — DISMISSED** [review]

**Comment by j-pendlebury** [comment]
@dr3 @sareh freshly rebased, can I get some approvals? 😄

**Review by dr3 — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]

**Pull request merged**